frontend/packages/dev-console/src/components/pipelineruns/logs/log-snippet-types.ts

```diff
@@ -0,0 +1,28 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { Condition, PLRTaskRunStep } from '../../../utils/pipeline-augment';
+
+type ErrorDetails = {
+  title: string;
+};
+
+export type ErrorDetailsWithLogName = ErrorDetails & {
+  containerName: string;
+  podName: string;
+};
+export type ErrorDetailsWithStaticLog = ErrorDetails & {
+  staticMessage: string;
+};
+
+export type CombinedErrorDetails = ErrorDetailsWithLogName | ErrorDetailsWithStaticLog;
+
+export type TaskRunStatus = {
+  completionTime?: string;
+  conditions: Condition[];
+  podName?: string;
+  startTime?: string;
+  steps?: PLRTaskRunStep[];
+};
+
+export type TaskRunData = K8sResourceKind & {
+  status: TaskRunStatus;
+};
```

Comment:
Failed to remove this code 😄  Please trim it out so we don't have confusion on which to use.

Reply:
yup.. updated..thanks